app/presenters/taxon_presenter.rb

```diff
@@ -12,14 +12,16 @@ class TaxonPresenter
     to: :taxon
   )
 
+  SUPERGROUPS = %w(guidance_and_regulation
+                   news_and_communications
+                   services).freeze
+
```

Comment:
The order of the sections should remain the same:
* Services
* Guidance and regulation
* News and communications

Reply:
Ok cool, I will add a comment to that effect